src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageEndpoint.cs

```diff
@@ -23,7 +23,7 @@ internal class RazorLanguageEndpoint : IRazorLanguageQueryHandler, IRazorMapToDo
         internal static readonly Range UndefinedRange = new Range(
             start: new Position(-1, -1),
             end: new Position(-1, -1));
-        private static readonly long UndefinedDocumentVersion = -1;
+        private static readonly int? UndefinedDocumentVersion = -1;
```

Comment:
If we're going to represent an undefined document version as -1 we don't need it to be `nullable` elsewhere